asa_parser/src/asa_parse.py

```diff
@@ -63,6 +63,10 @@ def show_tech_support_detail(self):
         """Parser for show cpu detailed"""
         return json.dumps({'text': self.get_show_section('tech-support detail')})
 
-    def show_context_details(self):
-        """Parser for show context details"""
```

Comment:
Why remove this? This was part of another teams sprint and should remain part of the codebase. Please add it back in and we can approve this pull request